src/server/dragonfly_test.cc

```diff
@@ -316,9 +324,14 @@ TEST_F(DflyEngineTest, Hello) {
               ErrArg("WRONGPASS invalid username-password pair or user is disabled."));
 
   resp = Run({"hello", "3", "AUTH", "default", ""});
+  ASSERT_THAT(resp, ErrArg("WRONGPASS invalid username-password pair or user is disabled."));
```

Comment:
Valkey does not allow AUTH user without password, now we got the correct semantics.